src/reconcile.rs

```diff
@@ -173,6 +241,26 @@ where
         );
         self.set_condition(condition).await
     }
+
+    /// A reconciler function to adsour finalizer to the list of finalizers.
```

Comment:
Typo:
 /// A reconciler function to add our finalizer to the list of finalizers.